streamx-plugin/streamx-flink-submit/src/main/scala/com/streamxhub/streamx/flink/submit/trait/KubernetesNativeSubmitTrait.scala

```diff
@@ -54,6 +55,8 @@ import scala.util.Try
 //noinspection DuplicatedCode
 trait KubernetesNativeSubmitTrait extends FlinkSubmitTrait {
 
+  lazy val workspace:Workspace = new Workspace(StorageType.LFS)
```

Comment:
Same as `streamx-plugin/streamx-flink-packer/src/test/scala/com/streamxhub/streamx/flink/packer/MavenToolSpec.scala`.
I actually prefer the more concise `WorkspaceGetter.local`